servers/lib/server/handlers/findteam.coffee

```diff
@@ -0,0 +1,55 @@
+koding  = require './../bongo'
+async   = require 'async'
+Tracker = require '../../../../workers/social/lib/social/models/tracker.coffee'
+{ protocol, hostname } = require 'koding-config-manager'
+emailsanitize = require '../../../../workers/social/lib/social/models/user/emailsanitize'
+
+module.exports = (req, res) ->
+
+  { email } = req.body
+  { JUser } = koding.models
+
+  return res.status(400).send 'Invalid email!'  unless email
+
+  queue = [
+    (next) ->
+      sanitizedEmail = emailsanitize email, { excludeDots: yes, excludePlus: yes }
+      JUser.one { sanitizedEmail }, (err, user) ->
+        return next err  if err
+        return next 'User not found'  unless user
```

Comment:
i think we can ignore the error here not to leak information ( but we still need to stop the execution )